tests/unittests/Starboard/PathMapperTests.mm

```diff
@@ -69,6 +71,8 @@
 
     CPathMapper mapper16("/AppSupport/.\\?/");
     EXPECT_EQ_MSG(0, wcscmp(mapper16, L"d:\\temp\\AppSupport\\+"), "%S", (const wchar_t*)mapper16);
```

Comment:
nit: for these guys, prefer `EXPECT_STREQ(expected, actual)`; gtest/taef will properly log the inputs and use the wchar_t overload of StrHelperCmpEQ